torch/onnx/symbolic_opset9.py

```diff
@@ -1663,6 +1663,40 @@ def gather(g, self, dim, index, sparse_grad=False):
     return g.op("ReduceSum", mul, axes_i=[dim], keepdims_i=0)
 
 
+@parse_args('v', 'is', 'b', 'i')
```

Comment:
Do we have b type annotation?

Reply:
from symbolic_helper, I can see that 'b' is for bool(tval)

Reply:
Just to confirm, since I was using my phone to do the review on Friday.